src/root/views/DrefApplicationForm/EventDetails/index.tsx

```diff
@@ -49,6 +54,32 @@ function EventDetails(props: Props) {
   } = props;
 
   const error = getErrorObject(formError);
+  const imagesValue = React.useMemo(() => (
+    value?.images_file?.map(d => d.id).filter(d => !!d) as number[] | undefined
```

Comment:
Lets not cast the output. Is image_files not properly typed?

Reply:
Looking at the code it looks like, the transformation here is from `(number | undefined)[]` to `number[]` which is achieved from the `filter`